sharding/node/backend.go

```diff
@@ -138,7 +136,11 @@ func (s *ShardEthereum) Close() {
 	defer s.lock.Unlock()
 
 	for kind, service := range s.services {
```

Comment:
1. when we stop the services, should we stop each in the reverse order we appended to `serviceTypes`? (i.e. does ordering matter when attempt to stop services)
2. if 1. is yes then we should use `for _, kind := range s.serviceTypes` for  `(s *ShardEthereum) Close() `

Reply:
I don't think it matters